src/components/contextual/pages/pool/PoolActionsCard.vue

```diff
@@ -36,6 +35,7 @@ const { networkSlug } = useNetwork();
  */
 const joinDisabled = computed(
   (): boolean =>
+    isDeprecatedPool.value ||
```

Comment:
Lets add `joinsDisabled?: boolean` to the DeprecatedDetails and use that. There may be some cases where we shouldn't block joins on deprecated pools.

Reply:
done